tests/dashbio_demos/app_clustergram.py

```diff
@@ -130,7 +135,28 @@ def layout():
                     label='About',
                     value='what-is',
                     children=html.Div(className='clustergram-tab', children=[
-                        html.P('Gene expression data and heatmap with dendrogram.')
+                        html.H3('What is Clustergram?'),
+                        html.P('Clustergram is a combination of a heatmap and '
+                               'dendrograms that allows you to display '
+                               'hierarchical clustering data. '
+                               'Clusters on the dendrograms are highlighted in '
+                               'different colors if they comprise data points '
```

Comment:
```suggestion
                               'similar colors if they comprise data points '
```
I'm not sure, but that's what I would expect logically, if/since they "share some minimal level of correlation."

Reply:
You're right -- good catch! The colors are actually the exact same (that's what you see in the app -- the different-colored "trees" correspond to different clusters. I'll reword this part to reflect that :)